### ISSUE TITLE
NPE (Columba-CCFinder)

### ISSUE THREAD

**Issue by dmazinanian** [tsantalis/JDeodorant#38]
Clone # 347 (Group 3550, Pair 1-2): CSVAddressbookImporter#importAddressbook (row 896) and org.columba.addressbook.folder.importfilter.MozillaCSVAddressbookImporter#importAddressbook (row 897)

```
java.lang.NullPointerException
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.extractClone(ExtractCloneRefactoring.java:557)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.apply(ExtractCloneRefactoring.java:333)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.checkFinalConditions(ExtractCloneRefactoring.java:4858)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.testRefactoring(Application.java:669)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.start(Application.java:185)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
```

**Comment by dmazinanian** [comment]
Same in Columba-CloneDR-Clone # 54 (Group 54, Pair 1-2): org.columba.addressbook.folder.importfilter.MozillaCSVAddressbookImporter#importAddressbook (row 593) and CSVAddressbookImporter#importAddressbook (row 594)

**Comment by dmazinanian** [comment]
Same in Columba-Deckard-Clone # 18 (Group 25, Pair 1-2)

**Comment by tsantalis** [comment]
Fixed in commit [4293485fc07778903b02622d2db2fc6ad74e5c7e](https://github.com/tsantalis/JDeodorant/commit/4293485fc07778903b02622d2db2fc6ad74e5c7e)
